packages/migrate/src/utils/getGithubIssueUrl.ts

```diff
@@ -69,7 +69,7 @@ export async function wouldYouLikeToCreateANewIssue(options: IssueOptions) {
       title: options.title ?? '',
       body: issueTemplate(platform, options),
     })
-    await open(url)
+    await open(url, { wait: true })
```

Comment:
[The README](https://github.com/sindresorhus/open/tree/v7.4.2#wait) is confusing since here we do not specify an app but this does something for you 🤔 

```
[](https://github.com/sindresorhus/open/tree/v7.4.2#wait)wait

Type: boolean
Default: false

Wait for the opened app to exit before fulfilling the promise. If false it's fulfilled immediately when opening the app.

Note that it waits for the app to exit, not just for the window to close.

On Windows, you have to explicitly specify an app for it to be able to wait.
```

Reply:
We should quickly try it on all platforms (Windows, macOS and linux) with the integration version

Reply:
No `app` is needed for the version of `open` that is installed (7.4.2)

Reply:
Hmm, why would `wait:true` be desired in our use case? We want it to open a browser tab or window and immediatly return in CLI.

Reply:
I believe the documentation on the `open` package is either outdated, wrong in general, or wrong for Windows/WSL only (which are the only OSes I can test on right now).
Before setting the `wait` flag, I went through the [source code](https://github.com/sindresorhus/open/blob/v7.4.2/index.js). It actually returns the control to the `cli`
 as soon as the browser tab is open. Without it, on Windows, the main Node.js process terminates before `open` forwards the request to open the default browser / specific browser to Windows' terminal shell.

Reply:
That sounds like worth a quick deeper look and then potentially a PR to `open`.
(Note that we this might be fixed in newer verisons of coure)

Reply:
`await open(url, { wait: true })` works with both the current version and the latest version of `open`, at least on my Windows 10 & WSL